docs/community_resources/translations.md

```diff
@@ -5,6 +5,54 @@ If you'd like to see eZ Platform in your language, you can contribute to the tra
 [`ezplatform-i18n`](https://github.com/ezsystems/ezplatform-i18n) contains the XLIFF files providing translations.
 You can use an XLIFF editor of your choice to contribute strings in your language.
 
+## How to translate the interface using Crowdin
+
+If you wish to contribute to an existing translation of Back Office or start a new one, the best way is to start with in-context translation (but you can also [translate directly on the Crowdin website](#translating-outside-in-context)).
```

Comment:
I'd split that into two sentences. The interface is important as well, even if we first mention in-context.
We can also reference the translation platform @SylvainGuittard has configured.